test/test.js

```diff
@@ -0,0 +1,25 @@
+const { reviewGenerator } = require('../data_generator/reviewGenerator')
+
+/**
+ * testing reviewGenerator
+*/
+test('Should return an empty array', () => {
+  expect(reviewGenerator(0)).toHaveLength(0);
+});
+
+test('Should create one review', () => {
+  expect(reviewGenerator(1)).toHaveLength(1);
+});
+
+test('Should generate a random name', () => {
+  expect(typeof reviewGenerator(1)[0].name).toBe('string');
+});
+
+test('Should generate a date matches formate YYYY-MMMM', () => {
```

Comment:
Hi, just curious why month represented by 4 digits?

Reply:
It means it will generate a month without abbreviation. The result will be 'April', 'October', etc/
It's a fixed formate in `npm casual` (a random data generator)